doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+The NumPy library contains multidimensional array and matrix data structures (you'll find more information about this in later sections). It provides **ndarray**, a homogeneous n-dimensional array object, with methods to efficiently operate on it. NumPy can be used to perform a wide variety of mathematical operations on arrays.  It adds powerful data structures to Python that guarantee efficient calculations with arrays and matrices and it supplies an enormous library of high-level mathematical functions that operate on these arrays and matrices. 
+
+`Learn more about NumPy here <https://docs.scipy.org/doc/numpy-1.17.0/user/whatisnumpy.html>`_!
+
+Installing NumPy
+----------------
+  
+To install NumPy, we strongly recommend using a scientific Python distribution. If you're looking for the full instructions for installing NumPy on your operating system, you can `find all of the details here <https://www.scipy.org/install.html>`_.
+
+
+  
+If you already have Python, you can install NumPy with
+
+::
+
+  conda install numpy
+  
+or 
+
+::
+
+  pip install numpy
+  
+If you don't have Python yet, you might want to consider using `Anaconda <https://www.anaconda.com/>`_. It's the easiest way to get started. The good thing about getting this distribution is the fact that you don’t need to worry too much about separately installing NumPy or any of the major packages that you’ll be using for your data analyses, like pandas, Scikit-Learn, etc.
+
+You can find all of the installation details in the `Installation <https://www.scipy.org/install.html>`_ section at scipy.org.
+
+How to import NumPy
+-------------------
+
+Any time you want to use a package or library in your code, you first need to make it accessible. 
+
+In order to start using NumPy and all of the functions available in NumPy, you'll need to import it. This can be easily done with this import statement:
+
+::
+
+  import numpy as np 
+
+(We shorten "numpy" to "np" in order to save time and also to keep code standardized so that anyone working with your code can easily understand and run it.)
+
+What’s the difference between a Python list and a NumPy array? 
+--------------------------------------------------------------
+  
+NumPy gives you an enormous range of fast and efficient numerically-related options. While a Python list can contain different data types within a single list, all of the elements in a NumPy array should be homogenous. The mathematical operations that are meant to be performed on arrays wouldn't be possible if the arrays weren't homogenous. 
```

Comment:
I don't think _numerically-related options_ is very clear. How about _ways of creating arrays and manipulating numerical data inside them_?

Reply:
The second sentence isn't quite correct. How about `wouldn't be possible` -> `would be extremely inefficient`?

Reply:
Got it!